libs/core/src/framework/types.ts

```diff
@@ -175,6 +176,7 @@ export type CommandMetadata<
   readonly auth: CommandHandler<Input, Output>[];
   readonly body: CommandHandler<Input, Output>;
   readonly secure?: boolean;
+  readonly authStrategy?: AuthStrategies;
```

Comment:
where's `authStrategy` being used within the framework?

Reply:
It's used in the `authenticate` function -> `adapters/src/trpc/index.ts`. This function is used by both the command and query functions.